Integrations/integration-ProtectWise_CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## [Unreleased]
+  * Fixed an issue where events were not fetched properly.
```

Comment:
release notes items should start with `-` and not `*`

Reply:
fixed